homeassistant/components/unifiprotect/button.py

```diff
@@ -73,6 +93,19 @@ class ProtectButtonEntityDescription(
 )
 
 
+async def _remove_adopt_button(
```

Comment:
```suggestion
@callback
def _async_remove_adopt_button(
```

This can be a callback now